### PR TITLE
refactor: improve readability of readmes

### PR DESCRIPTION
**Pull request by TsubasaK111**
@shunjikonishi review me! 😎

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
-It uses Thor. See detail in [Thor website](http://whatisthor.com/)
+## Requirements
 
-## How to get input parameters
-APP class has a static function `main`.
+- Ruby (For the version see our [docs](https://code-check.github.io/docs/en/reference_users/#serverside-language-and-tool-versions) .)
+- Thor. Do `gem install thor`. (For details, see their [official website](http://whatisthor.com/).)
+## Recieve Inputs
+
+In [app/app.rb](app/app.rb) is a class called `App`.
```
Comment (shunjikonishi) [inline README.md:12]: In ?

File: README.md [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
+Build your console application in `App`'s `self.main` method.  
 
 ``` ruby
-def self.main(args, options) 
-end
+class App
+  def self.main(args, options)
+    args.each { |arg|
+      # 以下の行を自分のコードと置換してね。
```
Comment (shunjikonishi) [inline README.md]: Need translation

**Comment by shunjikonishi** [comment]
@TsubasaK111 fix me

File: README.md (hunk 1 above)
Comment (TsubasaK111) [inline README.md:12]: Yes, "In"!
Japanese translation:
[app/app.rb](app/app.rb) **の中に** `App`というクラスがある。

**Comment by TsubasaK111** [comment]
@shunjikonishi review me (again, sorry!)

**Comment by shunjikonishi** [comment]
LGTM

**Pull request merged**